templates/corporate/case-studies/atolio-case-study.md

```diff
@@ -114,6 +114,7 @@ confusing.”
 ---
 
 Check out our guide on [using Zulip for business](/for/business/). You can also
-learn how Zulip is being used at the [iDrift AS](/case-studies/idrift/) company,
-and the [End Point Dev](/case-studies/end-point/) software consultancy, and the
-[GUT contact](/case-studies/gut-contact/) support agency.
+learn how Zulip is being used at the [Semsee](/case-studies/semsee/) InsurTech
```

Comment:
I think this is the one place where _Semsee_ appears before _InsurTech startup_. Other instances seem to be _InsurTech startup Semsee_. Might not be a big deal, but I wanted to point it out.

Reply:
Changed to match the rest, thanks!